kitsune/sumo/static/sumo/js/libs/jquery.autoresize.js

```diff
@@ -88,12 +88,12 @@
 
 		bind: function() {
 
-			var check = $.proxy(function(){
+			var check = $.bind(function(){
 				this.check();
 				return true;
 			}, this);
```

Comment:
It seems that `Function.prototype.bind()` should be called like this instead?
```suggestion
			var check = (function(){
				this.check();
				return true;
			}).bind(this);
```

Reply:
I believe you are correct.